src/chainrules.jl

```diff
@@ -2,6 +2,15 @@ ChainRulesCore.@scalar_rule(xlogx(x::Real), (1 + log(x),))
 ChainRulesCore.@scalar_rule(xlogy(x::Real, y::Real), (log(y), x / y,))
 ChainRulesCore.@scalar_rule(xlog1py(x::Real, y::Real), (log1p(y), x / (1 + y),))
 
+ChainRulesCore.@scalar_rule(
+    xexpx(x::Real),
+    (iszero(x) ? one(Ω) : isfinite(x) ? Ω / x * (1 + x) : Ω,)
```

Comment:
This is simply
```suggestion
    (xexpy(x + 1, x),)
```
If one wants to reuse part of the forward pass (`exp(x)`) one should implement `frule` and `rrule` explicitly but not divide the result by `x`. It's an optimization which we don't perform for `xlogy` etc. either, so I think `xexpy` is fine in this PR.

Reply:
What's wrong with `Ω / x`?